### PR TITLE
Update tenant-operator's integration config docs

### PR DESCRIPTION
**Pull request by ShahidHussain-srti**
Related changes in TO: https://github.com/stakater-ab/tenant-operator/pull/190

### CODE REVIEW

**Comment by stakater-user** [comment]
@LilShah Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-253-0017da9b`

File: content/sre/tenant-operator/integration-config.md [hunk 1]
```diff
@@ -9,78 +9,120 @@ metadata:
   name: tenant-operator-config
   namespace: stakater-tenant-operator
 spec:
+  nexus:
+    enabled: true
+    sso:
+      clientName: nexus3
```
Comment (abdulhaseeb2) [inline content/sre/tenant-operator/integration-config.md]: Please remove this. As this feature isn't implemented yet, so no need to mention it here.

File: content/sre/tenant-operator/integration-config.md [hunk 2]
```diff
@@ -9,78 +9,120 @@ metadata:
...
+    group:
+      labels:
         role: customer-reader
-      project:
-        workload-monitoring: 'true'
-      sandbox:
-        stakater.com/kind: sandbox
+    clusterAdminGroups:
+      - saap-cluster-admins
+      - stakater-team
```
Comment (abdulhaseeb2) [inline content/sre/tenant-operator/integration-config.md]: please remove this.

File: content/sre/tenant-operator/integration-config.md [hunk 3]
```diff
@@ -9,78 +9,120 @@ metadata:
...
-      group:
+    group:
+      labels:
         role: customer-reader
-      project:
-        workload-monitoring: 'true'
-      sandbox:
-        stakater.com/kind: sandbox
+    clusterAdminGroups:
+      - saap-cluster-admins
```
Comment (abdulhaseeb2) [inline content/sre/tenant-operator/integration-config.md]: rename this to cluster-admins

File: content/sre/tenant-operator/integration-config.md [hunk 4]
```diff
@@ -9,78 +9,120 @@ metadata:
...
+      - stakater-team
+    privilegedNamespaces:
+      - default
+      - ^openshift-*
+      - ^stakater-*
+      - ^kube-*
+      - ^redhat-*
+      - ^hive-*
```
Comment (abdulhaseeb2) [inline content/sre/tenant-operator/integration-config.md]: remove these

File: content/sre/tenant-operator/integration-config.md [hunk 5]
```diff
@@ -9,78 +9,120 @@ metadata:
...
+    privilegedServiceAccounts:
+      - ^system:serviceaccount:openshift-*
+      - ^system:serviceaccount:stakater-*
+      - ^system:serviceaccount:kube-*
+      - ^system:serviceaccount:redhat-*
+      - ^system:serviceaccount:hive-*
```
Comment (abdulhaseeb2) [inline content/sre/tenant-operator/integration-config.md]: remove these

File: content/sre/tenant-operator/integration-config.md [hunk 6]
```diff
@@ -9,78 +9,120 @@ metadata:
...
+      deny:
+        privilegedNamespaces:
+          users:
+            - system:serviceaccount:openshift-stakater-argocd:argocd-argocd-application-controller
+          groups:
+            - saap-cluster-admins
   rhsso:
     enabled: true
     endpoint:
+      url: https://iam-keycloak-openshift-stakater-auth.apps.binero-test.8sdzwd1l.kubeapp.cloud/
```
Comment (abdulhaseeb2) [inline content/sre/tenant-operator/integration-config.md]: why are you using our actual URL in documentation?

File: content/sre/tenant-operator/integration-config.md [hunk 7]
```diff
@@ -9,78 +9,120 @@ metadata:
...
       secretReference:
         name: auth-secrets
-        namespace: openshift-auth
-      url: >-
-        https://iam-keycloak-auth.apps.prod.abcdefghi.kubeapp.cloud/
+        namespace: stakater-auth
   vault:
     enabled: true
     endpoint:
+      url: "https://stakater-vault-stakater-vault.apps.binero-test.8sdzwd1l.kubeapp.cloud/"
```
Comment (abdulhaseeb2) [inline content/sre/tenant-operator/integration-config.md]: why are you using our actual URL in documentation?

File: content/sre/tenant-operator/integration-config.md [hunk 8]
```diff
@@ -9,78 +9,120 @@ metadata:
...
         openshift.io/node-selector: node-role.kubernetes.io/worker=
-    labels:
-      group:
+    group:
+      labels:
         role: customer-reader
-      project:
-        workload-monitoring: 'true'
-      sandbox:
-        stakater.com/kind: sandbox
```
Comment (abdulhaseeb2) [inline content/sre/tenant-operator/integration-config.md]: Why have you removed sandbox? Please add it back

File: content/sre/tenant-operator/integration-config.md [hunk 9]
```diff
@@ -9,78 +9,120 @@ metadata:
...
+      - ^system:serviceaccount:kube-*
+      - ^system:serviceaccount:redhat-*
+      - ^system:serviceaccount:hive-*
+    namespaceAccessPolicy:
+      deny:
+        privilegedNamespaces:
+          users:
+            - system:serviceaccount:openshift-stakater-argocd:argocd-argocd-application-controller
+          groups:
+            - saap-cluster-admins
```
Comment (abdulhaseeb2) [inline content/sre/tenant-operator/integration-config.md]: Again why are you using group names that stakater team manages? We don't want to show them on documentation, it should be abstract. Use `cluster-admins`

File: content/sre/tenant-operator/integration-config.md [hunk 10]
```diff
@@ -9,78 +9,120 @@ metadata:
...
+      - ^system:serviceaccount:openshift-*
+      - ^system:serviceaccount:stakater-*
+      - ^system:serviceaccount:kube-*
+      - ^system:serviceaccount:redhat-*
+      - ^system:serviceaccount:hive-*
+    namespaceAccessPolicy:
+      deny:
+        privilegedNamespaces:
+          users:
+            - system:serviceaccount:openshift-stakater-argocd:argocd-argocd-application-controller
```
Comment (abdulhaseeb2) [inline content/sre/tenant-operator/integration-config.md]: rename this to `system:serviceaccount:openshift-argocd:argocd-application-controller`

File: content/sre/tenant-operator/integration-config.md [hunk 11]
```diff
@@ -9,78 +9,120 @@ metadata:
...
   rhsso:
     enabled: true
     endpoint:
+      url: https://iam-keycloak-openshift-stakater-auth.apps.binero-test.8sdzwd1l.kubeapp.cloud/
       secretReference:
         name: auth-secrets
-        namespace: openshift-auth
-      url: >-
-        https://iam-keycloak-auth.apps.prod.abcdefghi.kubeapp.cloud/
+        namespace: stakater-auth
```
Comment (abdulhaseeb2) [inline content/sre/tenant-operator/integration-config.md]: rename this to `openshift-auth`

File: content/sre/tenant-operator/integration-config.md [hunk 12]
```diff
@@ -9,78 +9,120 @@ metadata:
...
   vault:
     enabled: true
     endpoint:
+      url: "https://stakater-vault-stakater-vault.apps.binero-test.8sdzwd1l.kubeapp.cloud/"
       secretReference:
         name: vault-root-token
-        namespace: vault
-      url: >-
-        https://vault.apps.prod.abcdefghi.kubeapp.cloud/
+        namespace: stakater-vault
```
Comment (abdulhaseeb2) [inline content/sre/tenant-operator/integration-config.md]: rename this to `vault`

File: content/sre/tenant-operator/integration-config.md [hunk 13]
```diff
@@ -9,78 +9,120 @@ metadata:
...
     sso:
-      accessorID: auth_oidc_aa6aa9aa
       clientName: vault
+      accessorID: "auth_oidc_058b5e09"
```
Comment (abdulhaseeb2) [inline content/sre/tenant-operator/integration-config.md]: rename the ID to a dummy ID like `auth_oidc_aa6aa9aa`

File: content/sre/tenant-operator/integration-config.md [hunk 14]
```diff
@@ -9,78 +9,120 @@ metadata:
...
+    labels:
+      role: customer-reader
+  sandbox:
+    labels:
+      stakater.com/kind: sandbox
   clusterAdminGroups:
-    - cluster-admins
-  managedNamespacePrefixes:
+    - saap-cluster-admins
+    - stakater-team
```
Comment (abdulhaseeb2) [inline content/sre/tenant-operator/integration-config.md]: remove these and use dummy values. Like `cluster-admins`

File: content/sre/tenant-operator/integration-config.md [hunk 15]
```diff
@@ -9,78 +9,120 @@ metadata:
...
-    - kube
-  managedServiceAccountPrefixes:
-    - 'system:serviceaccount:openshift'
-    - 'system:serviceaccount:stakater'
-    - 'system:serviceaccount:kube'
+    - ^openshift-*
+    - ^stakater-*
+    - ^kube-*
+    - ^redhat-*
+    - ^hive-*
```
Comment (abdulhaseeb2) [inline content/sre/tenant-operator/integration-config.md]: Remove these

File: content/sre/tenant-operator/integration-config.md [hunk 16]
```diff
@@ -9,78 +9,120 @@ metadata:
...
+  privilegedServiceAccounts:
+    - ^system:serviceaccount:openshift-*
+    - ^system:serviceaccount:stakater-*
+    - ^system:serviceaccount:kube-*
+    - ^system:serviceaccount:redhat-*
+    - ^system:serviceaccount:hive-*
```
Comment (abdulhaseeb2) [inline content/sre/tenant-operator/integration-config.md]: Remove these

File: content/sre/tenant-operator/integration-config.md [hunk 17]
```diff
@@ -9,78 +9,120 @@ metadata:
...
   namespaceAccessPolicy:
     deny:
-      managedNamespaces:
-        groups:
-          - cluster-admins
+      privilegedNamespaces:
         users:
-          - system:serviceaccount:openshift-argocd:argocd-application-controller
-          - adam@stakater.com
+          - system:serviceaccount:openshift-stakater-argocd:argocd-argocd-application-controller
```
Comment (abdulhaseeb2) [inline content/sre/tenant-operator/integration-config.md]: Use these instead

- system:serviceaccount:openshift-argocd:argocd-application-controller
- adam@stakater.com

File: content/sre/tenant-operator/integration-config.md [hunk 18]
```diff
@@ -9,78 +9,120 @@ metadata:
...
+        groups:
+          - saap-cluster-admins
```
Comment (abdulhaseeb2) [inline content/sre/tenant-operator/integration-config.md]: Use this instead
- cluster-admins

**Review by abdulhaseeb2 — CHANGES_REQUESTED** [review]
Please don't use the integrationConfig of binero-test in the documentation.

**Comment by stakater-user** [comment]
@LilShah Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-253-aec7db5d`

**Review by abdulhaseeb2 — APPROVED** [review]
LGTM

**Comment by stakater-user** [comment]
@LilShah Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.188`

**Pull request merged**